charts/redpanda/helpers.go

```diff
@@ -175,7 +175,7 @@ func TLSEnabled(dot *helmette.Dot) bool {
 func ClientAuthRequired(dot *helmette.Dot) bool {
 	listeners := []string{"kafka", "admin", "schemaRegistry", "rpc", "http"}
 	for _, listener := range listeners {
-		required := helmette.Dig(dot.Values.AsMap(), false, listener, "tls", "requireClientAuth")
+		required := helmette.Dig(dot.Values.AsMap(), false, "listeners", listener, "tls", "requireClientAuth")
```

Comment:
Oh dear this has been wrong for a while. I wonder how this didn't show up sooner? I'm guessing there's not too many tests that actually use this function.....

Reply:
I don't know if any test is asserting that Client Auth is configured.